packages/seedless-onboarding-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.3.0]
+
+### Uncategorized
```

Comment:
@himanshuchawla009 could you please add this changelog to the appropriate section?